apps/src/code-studio/components/ShareAllowedDialog.jsx

```diff
@@ -459,9 +479,22 @@ class ShareAllowedDialog extends React.Component {
                       )}
                     </span>
                   )}
+                  {selectedSong && (
```

Comment:
`selectedSong` is whether/what they've selected from the songs dropdown?

Reply:
yup!